pkgs/development/python-modules/mnist/default.nix

```
@@ -0,0 +1,26 @@
+{ buildPythonPackage, fetchPypi, isPy27, lib, mock, numpy }:
+
+buildPythonPackage rec {
+  pname = "mnist";
+  version = "0.2.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0m3vz9z7gay8nndid2784j95fzif502shfvik3f8mggvx6nlz4r4";
+  };
+
+  propagatedBuildInputs = [
+  numpy
+  ] ++ lib.optionals (isPy27) [ mock ];
```

```suggestion
  ] ++ lib.optional isPy27 mock;
```
I'm pretty sure that should only be a test dependency. You should make upstream aware of that.